llpc/test/shaderdb/relocatable_shaders/PipelineVsFs_16BitInput.pipe

```diff
@@ -0,0 +1,52 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% \
+; RUN:         -enable-relocatable-shader-elf \
+; RUN:         -o %t.elf %gfxip %s %s -v | FileCheck --match-full-lines -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: // LGC glue shader results
+; SHADERTEST: define amdgpu_vs {{.*}}
+; SHADERTEST: [[ld:%[0-9]+]] = call half @llvm.amdgcn.struct.tbuffer.load.f16({{.*}}) #1
+; SHADERTEST: [[cast:%[0-9]+]] = bitcast half [[ld]] to i16
+; SHADERTEST: [[zext:%[0-9]+]] = zext i16 [[cast]] to i32
+; SHADERTEST: [[result:%[0-9]+]] = bitcast i32 [[zext]] to float
```

Comment:
I'd like to confirm that we want to `zext` the floating point value instead of doing a `fpext`. Does it rely on the vertex side truncating this back in a similar way?

Reply:
This is the correct code.  We just want to pass the bits through.  The vertex shader will just truncate the bit.  This is the code that is generated:

```
  %1 = bitcast float %0 to <2 x half>
  %.i0 = extractelement <2 x half> %1, i64 0
```